_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -54,6 +56,35 @@ popular Bitcoin infrastructure software.
     problems that are the result of this bug---but developers are still
     investigating.
 
+- **US Patent 7,110,538 has expired:** Bitcoin transactions are secured using
+  [ECDSA][] (the _Elliptic Curve Digital Signature Algorithm_). Verifying
+  signatures involves multiplying points on the elliptic curve by scalars.
+  Typically, each transaction input requires one or more signature verifications,
+  meaning that syncing the Bitcoin block chain can require many millions of these
+  elliptic curve point multiplications. Any technique to make point
+  multiplications more efficient therefore has the potential to significantly
+  speed up Bitcoin's initial sync.
+
+    In 2011, [Hal Finney described][finney endomorphism] a way to more
+    efficiently compute elliptic curve point multiplications using an
+    [endomorphism][] on the curve (a mapping from the curve to itself which
+    preserves all relationships between points). By using this endomorphism,
+    the multiplication can effectively be broken into two parts, calculated separately,
```

Comment:
That's kind of true, but I'm not sure it's worth mentioning. If the multiplication was completely split up, it would be slower.

The idea is really that a single multiplication with a 256-bit number is split into two multiplications with 128-bit numbers, and those two are then computed *simultaneously* (which in practice means the same number of point additions, but only 128 point doublings instead of 256).